recipes/coin-clp/all/conanfile.py

```diff
@@ -49,7 +49,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("coin-utils/2.11.6", transitive_headers=True)
+        self.requires("coin-utils/2.11.6", transitive_headers=True, transitive_libs=True)
```

Comment:
We are trying to track these since they are the opposite of our default assumption so it's important to have a comment :)

```suggestion
        # Symbols are exposed https://github.com/conan-io/conan-center-index/pull/16053#issuecomment-1512637106
        self.requires("coin-utils/2.11.6", transitive_headers=True, transitive_libs=True)
```